test/client/jest.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
 		),
 	},
 	modulePaths: [ '<rootDir>/extensions' ],
+	moduleDirectories: [ 'node_modules', '<rootDir>/test-helpers/' ],
```

Comment:
Required to be able to import the customer render function like this `import { render } from 'config/testing-library';`